### ISSUE TITLE
Confused about the reason for refactoring Dataloader code

### ISSUE THREAD

**Issue by wyf0912** [thstkdgus35/EDSR-PyTorch#236]
Thanks for your contribution.

According to my understand, the refactored dataloader can add the scale label. But the scale label can be returned in __getitem__. In addation, I think it is thread-unsafe to set scale in _ms_loop

Looking forward to your repy.
